src/torchmetrics/functional/multimodal/clip_iqa.py

```diff
@@ -329,3 +331,20 @@ def clip_image_quality_assessment(
         anchors = _clip_iqa_get_anchor_vectors(model_name_or_path, model, processor, prompts_list, device)
         img_features = _clip_iqa_update(model_name_or_path, images, model, processor, data_range, device)
         return _clip_iqa_compute(img_features, anchors, prompts_names)
+
+
+if TYPE_CHECKING:
+    from functools import partial
+    from typing import Any, cast
+
+    images = cast(Any, None)
+
+    f = partial(clip_image_quality_assessment, images=images)
+    f(prompts=("colorfullness",))
+    f(
+        prompts=("quality", "brightness", "noisiness"),
+    )
+    f(
+        prompts=("quality", "brightness", "noisiness", "colorfullness"),
+    )
+    f(prompts=(("Photo of a cat", "Photo of a dog"), "quality", ("Colorful photo", "Black and white photo")))
```

Comment:
```suggestion
```
I assume these are more like testing? should they really be in the code?

Reply:
that are some good arguments...
Alright, I mark these as resolved for now, and then we should probably come up with a better process for including testing of types in our codebase